core/src/main/java/ai/timefold/solver/core/config/heuristic/selector/move/NearbyUtil.java

```diff
@@ -162,7 +162,7 @@ private static ValueSelectorConfig configureSecondaryValueSelector(ValueSelector
                 .withValueSelectorConfig(valueConfig);
     }
 
-    private static String addRandomSuffix(String name, Random random) {
+    public static String addRandomSuffix(String name, Random random) {
```

Comment:
`NearbyUtil`... seems like this class should have been in Enterprise, we just didn't notice it.

Reply:
Other non-enterprise classes use this class. I believe we cannot move it to the enterprise repo without breaking things, and although it is related to nearby feature, its logic aligns with the solver repo as we try to enable the nearby feature from the solver logic.